cms/static/js/views/paging.js

```diff
@@ -0,0 +1,143 @@
+;(function (define) {
```

Comment:
This file used to be exactly here, but I moved it into common/components as part of my previous story. Since you are moving it back, it would be good to not lose the history.